core/api.raml

```diff
@@ -0,0 +1,43 @@
+#%RAML 1.0 Library
```

Comment:
Unfortunately RAML appears not to allow nested `ResourceType`, would like to split these up into separate files.